src/main/java/Duke.java

```diff
@@ -1,10 +1,111 @@
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm KaiKai.");
+        System.out.println("What can I do for you?");
+        System.out.println("______________________________________");
+
+        Scanner sc = new Scanner(System.in);
+        String str = sc.nextLine();
+
+
+        int taskNum;
+
+        while (!str.equals("bye")) {
+            if (str.equals("list")) {
+                printTasks();
+            } else if (str.equals("mark")) {
+                printTasks();
+                System.out.println("Which task would you like to mark as completed?");
+                taskNum = sc.nextInt();
+                sc.nextLine();
+                if (taskNum > numTasks || taskNum <= 0) {
+                    System.out.println("Uh oh! It seems like that task doesn't exist!");
+                } else {
+                    tasks[taskNum - 1].setDone(true);
+                    System.out.println("Nice! I've marked this task as done: ");
+                    tasks[taskNum - 1].printTask();
+                }
+            } else if (str.equals("unmark")) {
+                printTasks();
+                System.out.println("Which task would you like to mark as incomplete?");
+                taskNum = sc.nextInt();
+                sc.nextLine();
+                if (taskNum > numTasks || taskNum <= 0) {
+                    System.out.println("Uh oh! It seems like that task doesn't exist!");
+                } else {
+                    tasks[taskNum - 1].setDone(false);
+                    System.out.println("Okie, I've marked this task as not done yet:");
+                    tasks[taskNum - 1].printTask();
+                }
+            } else if (str.equals("add")) {
+                System.out.println("Okie, what type of task is this?");
+                System.out.println("1.Todo");
+                System.out.println("2.Deadline");
+                System.out.println("3.Event");
+                str = sc.nextLine();
+                switch (str) {
+                case "1":
+                    addTodo();
+                    break;
+                case "2":
+                    addDeadline();
+                    break;
+                case "3":
+                    addEvent();
+                    break;
+                default:
+                    System.out.println("Uh oh, please enter a valid number!");
+                }
+                System.out.println("______________________________________");
+            }
+            else{
+                System.out.println("Sorry, I don't recognise that command. Please try again!");
+                System.out.println("______________________________________");
+            }
+            str = sc.nextLine();
+        }
+        System.out.println("Bye! Hope to see you again!");
+    }
+
+    public static void printTasks(){
+        for (int i=0; i<numTasks; i++){
```

Comment:
I think you also might need more spaces here? This is in reference to "White space within a statement" in the coding standards